src/librustdoc/clean/mod.rs

```diff
@@ -3768,11 +3768,13 @@ pub enum ImplPolarity {
     Negative,
 }
 
-impl Clean<ImplPolarity> for hir::ImplPolarity {
+impl Clean<ImplPolarity> for ty::ImplPolarity {
     fn clean(&self, _: &DocContext<'_>) -> ImplPolarity {
         match self {
-            &hir::ImplPolarity::Positive => ImplPolarity::Positive,
-            &hir::ImplPolarity::Negative => ImplPolarity::Negative,
+            &ty::ImplPolarity::Positive |
+            // FIXME: do we want to do something else here?
```

Comment:
Almost certainly yes -- I think this means that reservation impls will show up in rustdoc as if they were real impls, which seems bound to lead to confusion, no?

Reply:
Is it enough to show the reservation attribute?

Reply:
cc @QuietMisdreavus (I think you know Rustdoc?)